### DISCUSSION TITLE
why are there duplicate settings for disabling jit?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1863] [Q&A] [answered]
why are there 2 duplicate settings for disabling jit?

![Screenshot_20220308-042944-1](https://user-images.githubusercontent.com/95953932/157132581-a8c1d929-fe8c-423a-bb3f-701a6db22179.jpg)
![Screenshot_20220308-042939-1](https://user-images.githubusercontent.com/95953932/157132606-f0ba9b12-e7dc-40dc-959d-454bc2a72250.jpg)

**Comment by csagan5** [comment, marked as answer]
The one in your first screenshot is from Privacy Settings, the second one is from Site Settings. Since the latter can also be set as the default for all we might not need the global one; @uazo what do you think?

**Reply by uazo** [reply]
> what do you think?

I agree, the former (Privacy Settings) can be eliminated

**Reply by csagan5** [reply]
I will remove it.
